index.html

```diff
@@ -0,0 +1,226 @@
+    <header class="mob">
+      <a class="ham">
+        <div id="toggle-icon">
+          <div class="ham-bar1"></div>
+          <div class="ham-bar2"></div>
+          <div class="ham-bar3"></div>
+        </div>
+      </a>
+      <nav class="header2">
+        <div>
+          <a href="./index.html">
+            <h2 class="header2-content-mob">Home Service</h2>
+          </a>
+        </div>
+        <div>
+          <a href="./about.html">
+            <h2 class="header2-content-mob">About Af</h2>
+          </a>
+        </div>
+      </nav>
+    </header>
+    <section class="pink-section">
+      <section>
+        <div class="banner-desk">
+          <img src="images/massage-bg.png" class="bg-img" alt="massage" />
+        </div>
+        <div class="home-text-container">
+          <div class="before-title">
+            No time to get to the beauty salon, ladies? No problem!
+          </div>
+          <h1 class="main-title">Hair, Skin and Nails treatment</h1>
+          <p class="short-description">
+            Book highly-qualified therapist to your door... with a working
+            experience of over 2 decades in Mauritius, Reunion, France and UK
+            combined.
+          </p>
+          <h2 class="perfect-solution">
+            Af Esthétique gives you your perfect solution
+          </h2>
+          <h3 class="like-us">
+            ‘Like’ us on Facebook for all the latest products, news and offers
+          </h3>
+        </div>
+      </section>
+      <section class="main-benefits">
+        <h2 class="benefits-title">Main Benefits</h2>
+        <div class="bar"></div>
+        <ul class="benefits-container">
+          <li>
+            <div class="benefits-small-container">
+              <i class="fa fa-heartbeat benefit-icons" aria-hidden="true"></i>
+              <h3 class="benefits-small-container-title">Relaxation</h3>
+              <p class="benefits-small-container-info">
+                Every service falling under a spa inclusive of a massage and
+                body work can greatly aid you in relaxing. It's one of the best
+                ways to manage stress.
+              </p>
+            </div>
+          </li>
+          <li>
+            <div class="benefits-small-container">
+              <i class="fa fa-child benefit-icons" aria-hidden="true"></i>
+              <h3 class="benefits-small-container-title">
+                Improved Self Esteem and Confidence
+              </h3>
+              <p class="benefits-small-container-info">
+                It is a proven fact that taking adequate time to pamper oneself
+                and to taking good care of your body and mind can greatly
+                improve your body image.
+              </p>
+            </div>
+          </li>
+          <li>
+            <div class="benefits-small-container">
+              <i class="fa fa-bed benefit-icons" aria-hidden="true"></i>
+              <h3 class="benefits-small-container-title">
```

Comment:
- [ ] I think you have missed this button `Join the CC Global Summit 2015` given in the screenshot In the `main benefit` section. You can name anything you want like `Join our SPA` but make sure you're following the design.

### Mobile version

### Behance design

![Screenshot from 2021-07-09 20-55-14](https://user-images.githubusercontent.com/22434660/125102828-586dc700-e0f9-11eb-8328-7f05b221c456.png)